src/Bookings.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
A very minor problem, but normally we only use capital letters in the first letter for React components. Everything else we name like a normal variable, so the first letter is not capitalised.

Could you fix this so it follows that *convention*?